manifests/init.pp

```diff
@@ -193,7 +193,7 @@
     include postfix::mta
   }
 
-  if $satellite {
+  if $satellite or $relayhost {
     if $mta {
       fail('enabling both the $mta and $satellite parameters is not supported. Please disable one.')
```

Comment:
Error message should be updated since it can now be triggered if `$relayhost` is set too.